src/pacer.js

```diff
@@ -644,7 +644,7 @@ let PACER = {
     //  attachment number of the document.
 
     let row = anchor.parentNode.parentNode;
-    if (row.childElementCount < 3) {
+    if (row.childElementCount <= 3) {
```

Comment:
Should we add a comment here to explain the different number of elements that are possible?

Reply:
Sure.